src/GrammarParser.php

```diff
@@ -13,6 +27,31 @@ class GrammarParser
     protected $parser = null;
 
     public function __construct()
+    {
+        $this->addDefaultPlugins();
+        $this->loadPlugins();
+    }
+
+    public function addDefaultPlugins()
+    {
+        // Note: load order does matter
```

Comment:
At first I sorted them alphabetically but then the test `\ParserGenerator\Tests\Extension\LookaheadTest::testErrorTrack` failed: `TextNode` needed to be loaded before `Lookahead`.

I then decided to simply keep the original order from the `require_once` directives from the end of the file.